src/utils.zig

```diff
@@ -8,6 +8,7 @@ const NeverFailingAllocator = main.heap.NeverFailingAllocator;
 
 pub const file_monitor = @import("utils/file_monitor.zig");
 pub const VirtualList = @import("utils/virtual_mem.zig").VirtualList;
+pub const meta = @import("utils/meta.zig");
```

Comment:
I think this is useful enough to have it in the top-level.
I'd suggest to make a separate PR first that moves relevant stuff (e.g. function pointer casts) there.